### ISSUE TITLE
 change labelText and hintText

### ISSUE THREAD

**Issue by Lalosantoyo51** [SimformSolutionsPvtLtd/flutter_credit_card#9]
example 
```dart
InputDecoration(
                  border: OutlineInputBorder(),
                  labelText: 'Card number',
                  hintText: 'xxxx xxxx xxxx xxxx',
                );
InputDecoration(
                  border: OutlineInputBorder(),
                  labelText: 'Numero de tarjeta',
                  hintText: 'xxxx xxxx xxxx xxxx',
                )
````

**Comment by BirjuVachhani** [comment]
@Lalosantoyo51 More description would be great!

**Comment by DevarshRanpara** [comment]
Will be Fixed in #22
